### PR TITLE
DNX documentation: Fixed several grammar and spelling mistakes.

### PR DESCRIPTION
**Pull request by bendono**
Fixed several grammar and spelling mistakes.

### CODE REVIEW

File: Documentation/dnx-instructions.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ DNX is the ".NET Execution Environment". It's a concept that came out of the ASP
 - It is console-everything.
 - It is the base mechanism and set of concepts on which ASP.NET 5 is built.
 
-You can have more than one DNX installed (under `.dnx`) on a machine will have a single one selected for use at any time. You use [DNVM](https://github.com/aspnet/dnvm), the .NET SDK  Manager, to acquire and manage DNX instances.
+You can have more than one DNX installed (under `.dnx`) on a machine and have a single one selected for use at any time. You use [DNVM](https://github.com/aspnet/dnvm), the .NET SDK  Manager, to acquire and manage DNX instances.
```
Comment (richlander) [inline Documentation/dnx-instructions.md]: How about "... on a machine and a single one will be selected for ..."

File: Documentation/dnx-instructions.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ You need one extra file for the demo to work, which is project.json. Add it besi
 }
 ```
 
-You need to restore any dependent nuget packages, with the dnu tool (comes with DNX). DNU will pull down NuGet packages from nuget.org or any other configured NuGet feed, such as MyGet.
+You need to restore any dependent nuget packages with the dnu tool (comes with DNX). DNU will pull down NuGet packages from nuget.org or any other configured NuGet feed, such as MyGet.
```
Comment (richlander) [inline Documentation/dnx-instructions.md]: How about we remove "any" in the first sentence.

File: Documentation/dnx-instructions.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ You can now run the app, with dnx.
 Switching to CoreCLR
 ====================
 
-For the purpose of this demo, you need an x64 DNX hosted on (actually including) CoreCLR. The CoreCLR build only generates X64 artifacts currently. You can install the latest version of  an X64 CoreCLR DNX with DNVM. You'll notice that the install process will crossgen the managed assemblies for you. Performance!
+For the purposes of this demo, you need an x64 DNX hosted on (actually including) CoreCLR. The CoreCLR build only generates X64 artifacts currently. You can install the latest version of an X64 CoreCLR DNX with DNVM. You'll notice that the install process will crossgen the managed assemblies for you. Performance!
```
Comment (richlander) [inline Documentation/dnx-instructions.md]: Can you explain this change?

**Comment by richlander** [comment]
LGTM modulo comments.

**Comment by bendono** [comment]
I had some difficulties merging this. Will close and retry.

**Pull request closed without merging**